### PR TITLE
fix: example -updates the examples to the new llama 3.2 3B instruct

### PR DESCRIPTION
**Pull request by Abiji-2020**
### Which issue(s) does this pull-request address?

Closes:  #47 

### Description

The last PR #51  contains many errors due to the standards of the commit message, which was updated in this branch and proper naming is followed. 

### Checklist

- [x] I have read the [contributor guide](https://github.com/i-am-bee/bee-agent-framework/blob/main/CONTRIBUTING.md)
- [x] Linting passes: `yarn lint` or `yarn lint:fix`
- [x] Formatting is applied: `yarn format` or `yarn format:fix`
- [x] Unit tests pass: `yarn test:unit`
- [x] E2E tests pass: `yarn test:e2e`
- [ ] Tests are included 
- [ ] Documentation is changed or added
- [x] Commit messages and PR title follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

**Comment by Abiji-2020** [comment]
@Tomas2D this is a new PR for the issue #47  as the I had messed up the old PR on the naming issues

**Comment by Abiji-2020** [comment]
@Tomas2D any update on the PR?

File: examples/agents/bee_advanced.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ import { z } from "zod";
 Logger.root.level = "silent"; // disable internal logs
 const logger = new Logger({ name: "app", level: "trace" });
 
-const llm = BAMChatLLM.fromPreset("meta-llama/llama-3-1-70b-instruct");
+const llm = BAMChatLLM.fromPreset("meta-llama/Llama-3.2-3B-Instruct");
```
Comment (Tomas2D) [inline examples/agents/bee_advanced.ts:25]: This model does not exist in the BAM environment.

File: examples/agents/sql.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ import { FrameworkError } from "bee-agent-framework/errors";
 import { UnconstrainedMemory } from "bee-agent-framework/memory/unconstrainedMemory";
 
 const llm = new GroqChatLLM({
-  modelId: "llama3-70b-8192",
+  modelId: "llama3.2",
```
Comment (Tomas2D) [inline examples/agents/sql.ts:9]: Verify whether the given model exists in GROQ.

File: examples/llms/providers/watsonx.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@ import "dotenv/config";
 import { BaseMessage } from "bee-agent-framework/llms/primitives/message";
 import { WatsonXChatLLM } from "bee-agent-framework/adapters/watsonx/chat";
 
-const chatLLM = WatsonXChatLLM.fromPreset("meta-llama/llama-3-1-70b-instruct", {
+const chatLLM = WatsonXChatLLM.fromPreset("meta-llama/Llama-3.2-3B-Instruct", {
```
Comment (Tomas2D) [inline examples/llms/providers/watsonx.ts:5]: The name is incorrect. The correct one should be `llama-3-2-3b-instruct`.
Please do a test run to ensure whether it works.

File: examples/llms/providers/watsonx_verbose.ts [hunk 4]
```diff
@@ -22,7 +22,7 @@ const template = new PromptTemplate({
 });
 
 const llm = new WatsonXLLM({
-  modelId: "meta-llama/llama-3-1-70b-instruct",
+  modelId: "meta-llama/Llama-3.2-3B-Instruct",
```
Comment (Tomas2D) [inline examples/llms/providers/watsonx_verbose.ts:25]: The name is incorrect. The correct one should be `llama-3-2-3b-instruct`.
Please do a test run to ensure whether it works.

File: src/adapters/bam/chatPreset.ts [hunk 5]
```diff
@@ -78,6 +78,21 @@ export const BAMChatLLMPreset = {
       },
     };
   },
+  "meta-llama/Llama-3.2-3B-Instruct": (): BAMChatLLMPreset => {
```
Comment (Tomas2D) [inline src/adapters/bam/chatPreset.ts:81]: Same as above.

File: package.json [hunk 6]
```diff
@@ -12,7 +12,7 @@
     "Bee Agent Framework",
     "NodeJS Agent Framework"
   ],
-  "packageManager": "yarn@4.1.1",
+  "packageManager": "yarn@4.5.0+sha512.837566d24eec14ec0f5f1411adb544e892b3454255e61fdef8fd05f3429480102806bac7446bc9daff3896b01ae4b62d00096c7e989f1596f2af10b927532f39",
```
Comment (Tomas2D) [inline package.json:15]: Don't change the version of the package manager. This should not be done in this PR.

File: src/adapters/shared/llmChatTemplates.ts [hunk 7]
```diff
@@ -86,6 +86,20 @@ const llama3: LLMChatTemplate = {
 
 {{system}}<|eot_id|>{{/system}}{{#user}}<|start_header_id|>user<|end_header_id|>
 
+{{user}}<|eot_id|>{{/user}}{{#assistant}}<|start_header_id|>assistant<|end_header_id|>
+{{assistant}}<|eot_id|>{{/assistant}}{{/messages}}<|start_header_id|>assistant<|end_header_id|>
```
Comment (Tomas2D) [inline src/adapters/shared/llmChatTemplates.ts:90]: I assume the formatting is wrong.

Comment (Tomas2D) [reply src/adapters/shared/llmChatTemplates.ts:90]: Why are you editing llama3?

File: src/adapters/shared/llmChatTemplates.ts [hunk 8]
```diff
@@ -86,6 +86,20 @@ const llama3: LLMChatTemplate = {
...
+  parameters: {
+    stop_sequence: ["<|eot_id|>"],
+  },
+};
+const llama32: LLMChatTemplate = {
+  template: new PromptTemplate({
+    schema: templateSchemaFactory(["system", "user", "assistant" as const]),
+    template: `{#messages}}{{#system}}<|begin_of_text|><|start_header_id|>system<|end_header_id|>
+{{system}}<|eot_id|>{{/system}}{{#user}}<|start_header_id|>user<|end_header_id|>
 {{user}}<|eot_id|>{{/user}}{{#assistant}}<|start_header_id|>assistant<|end_header_id|>
```
Comment (Tomas2D) [inline src/adapters/shared/llmChatTemplates.ts:103]: I assume the formatting is wrong.

File: src/adapters/watsonx/chatPreset.ts [hunk 9]
```diff
@@ -68,6 +68,22 @@ export const WatsonXChatLLMPreset = {
   "meta-llama/llama-3-8b-instruct"() {
     return this["meta-llama/llama-3-70b-instruct"]();
   },
+  "meta-llama/Llama-3.2-3B-Instruct": (): WatsonXChatLLMPreset => {
```
Comment (Tomas2D) [inline src/adapters/watsonx/chatPreset.ts:71]: Wrong name.

**Review by Tomas2D — CHANGES_REQUESTED** [review]

**Comment by Tomas2D** [comment]
I am closing this for being stale.

**Pull request closed without merging**